### ISSUE TITLE
Events doesn't fire on Ubuntu

### ISSUE THREAD

**Issue by condolent** [twitch4j/twitch4j#131]
### Prerequisites
* [x] If this is a question/suggestion, I have already considered discuss it on [Discord Server](https://discord.gg/FQ5vgW3)
* [x] This issue specifically has something to do with Twitch4J
* [x] I have attempted to [look for similar issues](https://github.com/twitch4j/twitch4j/issues) already

### Description
ChannelGoLiveEvent and such will not fire on linux systems. However when running on windows it works as expected!

### Steps to Reproduce
Basically just run it on ubuntu.

---

**Expected behavior:** 
Event `ChannelGoLiveEvent` and `ChannelGoOfflineEvent` fires when a tracked channel goes live/offline.

**Actual behavior:** 
Nothing happens.

**Stacktrace (if applicable):** 
No errors or anything helpful printed.

**Version affected:** 
1.0.0-alpha.24
